tendrl_node_agent/monitoring/plugins/handle_collectd_notification.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python
+import sys
+from subprocess import check_output
+import socket
+import json
+
+def getNotification():
+    notification_dict = {}
+    isEndOfDictionary = False
```

Comment:
As dicussed, check this logic again. Also remove 'isEndOfDictionary' if not required